src/actions/tabs.action-creator.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+import { SWITCH_TAB, } from './types';
+
+export function switchTabActionCreator(selectedTab){
```

Comment:
After I refactored the menu I can remove this since I am now communication from child to parent. Originally I had the menu in a sub-component, so I was communicating from child to parent and back to child. 
 
I'll see if I can ask the navigator to change the tab directly rather than sending a message through flux/redux.

Also, this paragraph says flux is the recommended way to communicate between components even if you aren't just passing state (unless I misunderstood this paragraph).  

"For communication between two components that don't have a parent-child relationship, you can set up your own global event system. Subscribe to events in componentDidMount(), unsubscribe in componentWillUnmount(), and call setState() when you receive an event. Flux pattern is one of the possible ways to arrange this."

https://facebook.github.io/react/tips/communicate-between-components.html